dto/src/main/java/org/jboss/pnc/dto/response/Edge.java

```diff
@@ -32,10 +34,19 @@
 @JsonDeserialize(builder = Edge.Builder.class)
 public class Edge<T> {
 
+    /**
+     * Source vertex name.
+     */
     private final String source;
 
+    /**
+     * Target vertex name.
+     */
     private final String target;
 
+    /**
+     * Edge cost.
+     */
     private final int cost;
```

Comment:
I don't understand this entity and the javadocs doesn't help. Improve the javadocs for the whole entity, please.

Reply:
@matejonnet Can you please provide the info I should write there?